File: UI/pc-part-picker/src/model/FormfactorType.ts
import { Motherboard } from "./Motherboard";
import { Pccase } from "./Pccase";
import { Powersupply } from "./Powersupply";

export interface FormfactorType {
    formfactorId: number;
    type: string | null;
    motherboards: Motherboard[];
    pccases: Pccase[];
    powersupplies: Powersupply[];
}

File: API/PCPartPicker/Models/FormfactorType.cs
using System;
using System.Collections.Generic;

namespace PCPartPicker.Models;

public partial class FormfactorType
{
    public int FormfactorId { get; set; }

    public string? Type { get; set; }

    public virtual ICollection<Motherboard> Motherboards { get; set; } = new List<Motherboard>();

    public virtual ICollection<Pccase> Pccases { get; set; } = new List<Pccase>();

    public virtual ICollection<Powersupply> Powersupplies { get; set; } = new List<Powersupply>();
}
